CustomScript/HandlerManifest.json

```diff
@@ -2,11 +2,11 @@
   {
     "version": 1.0,
     "handlerManifest": {
-      "disableCommand": "customscript.py -disable",
-      "enableCommand": "customscript.py -enable",
-      "installCommand": "customscript.py -install",
-      "uninstallCommand": "customscript.py -uninstall",
-      "updateCommand": "customscript.py -update",
+      "disableCommand": "./shim.sh -disable",
```

Comment:
I would drop the use of ./.  The agent constructs the path as necessary, so there's no need to be explicit.

Reply:
Oh perfect. Will remove. Thanks